docs/data-layer.md

```diff
@@ -0,0 +1,178 @@
+# Data Layer
+
+## Redux
+
+In eHoldings, data is connected on `routes` only. This is done via
+`react-redux`'s `connect` decorator.
+
+``` javascript
+connect(({ eholdings: { data } }) => ({
+  // ...mapStateToProps
+  // data needed by the component is accessed here
+}), {
+  // ...mapDispatchToProps
+  // actions used to send requests for data are defined here
+});
+```
+
+When data is needed, you must first request it with one of the actions
+defined in `mapDispatchToProps`. The data received in
+`mapStateToProps` will then update with a pending request, and update
```

Comment:
`will then update with a pending request, and update` minor nit but if you split this up it would be `The data received... will then update` and `The data recieved... and update again` so you should bridge these by subbing `and` for `then` or doing something weird like `will update with a pending request and, upon resolution of said request, will update once more`.